app/jobs/technical_metadata_job.rb

```diff
@@ -5,10 +5,10 @@ class TechnicalMetadataJob < ApplicationJob
   queue_as :default
 
   # @param [String] druid
-  # @param [Array<String>] filepaths of files
+  # @param [Hash<String, String>] map of filepaths of files to filenames
   # @param [Boolean] force even if md5 match
-  def perform(druid:, filepaths:, force: false)
-    errors = TechnicalMetadataGenerator.generate(druid: druid, filepaths: filepaths, force: force)
+  def perform(druid:, filepath_map:, force: false)
```

Comment:
Now accepts of map of filepaths to filenames instead of just a list of filepaths.